### ISSUE TITLE
Cannot tab to Readonly cells when in edit mode

### ISSUE THREAD

**Issue by ghulamghousdev** [bryntum/support#10213]
[Forum post](https://forum.bryntum.com/viewtopic.php?f=51&t=30724&p=155966#p155966)

 Tab to an editable cell item, when you highlight an editable cell by pressing enter you can only then cycle through editable cells (cant cycle through non-editable cells) unless you back tab to the first editable cell you selected
